### PR TITLE
Hypo to clipboard

### LINKED ISSUE #2466: "create a hypo to json exporter"
^ title are the words that Zamiel used to describe what I believe was being discussed on Discord today.

I can try and describe the process I'm envisioning:

- Play through a hypo of a certain seed (on the website)
- Somehow create a permanent record of the hypo; Zam mentioned it may be possible to make it exportable it as a .json, which then could be shared + viewed by others.

And for those curious, the larger aim of this would be to aid in a large-scale data collection effort

### PR DESCRIPTION
**Pull request by DrAnax**
Closes #2466

### CODE REVIEW

File: packages/client/src/lobby/createReplayJSON.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    globals.state.replay.hypothetical === null
+  ) {
+    chat.addSelf(
+      '<span class="red">Error:</span> You can only use that command during the review of a hypothetical.',
+      room,
+    );
+    return;
+  }
+  // Anonymize the players
+  const fakePlayers = ["Alice", "Bob", "Cathy", "Donald", "Emily", "Frank"];
```
Comment (Zamiell) [inline packages/client/src/lobby/createReplayJSON.ts:26]: shouldn't this be a constant somewhere, probably in @hanabi/data

Comment (DrAnax) [reply packages/client/src/lobby/createReplayJSON.ts:26]: That's a good idea (btw is 'Frank' correct? I kinda guessed)

Comment (Zamiell) [reply packages/client/src/lobby/createReplayJSON.ts:26]: https://github.com/Hanabi-Live/hanabi-live/commit/d443c1f717f5a282d02154bde231b3f7d0f63e80

**Comment by rogers-j** [comment]
Remove the extra `p` in 'clipboard'

File: packages/client/src/lobby/createReplayJSON.ts [hunk 2]
```diff
@@ -22,10 +23,16 @@ export default function createJSONFromReplay(room: string) {
     return;
   }
 
+  let playerNames: string[] = [];
+  if (HYPO_PLAYER_NAMES.length >= globals.metadata.playerNames.length) {
```
Comment (Zamiell) [inline packages/client/src/lobby/createReplayJSON.ts:27]: is the if statement necessary?

Comment (DrAnax) [reply packages/client/src/lobby/createReplayJSON.ts:27]: Yes because the linter complains

Comment (Zamiell) [reply packages/client/src/lobby/createReplayJSON.ts:27]: with what error

Comment (DrAnax) [reply packages/client/src/lobby/createReplayJSON.ts:27]: It doesn't anymore :S

**Comment by Zamiell** [comment]
nice work! this is a neat feature

**Pull request merged**